cardano-cli/test/Test/CLI/Shelley/Golden/Genesis/InitialTxin.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications #-}
```

Comment:
Are we using `TypeApplications` here? Same question for the other test modules introduced in this PR.